.travis.yml

```diff
@@ -1,40 +1,26 @@
-language: c
+language: generic
 os: linux
-sudo: required
-matrix:
-  include:
-    - os: linux
-      addons:
-        apt:
-          sources:
-            - ubuntu-toolchain-r-test
-            - george-edison55-precise-backports
-          packages:
-            - gcc-7
-            - gfortran-7
-            - g++-7
-            - cmake-data
-            - cmake
-      env:
-        - MATRIX_EVAL="FC=gfortran-7 && CC=gcc-7 && CXX=g++-7"
-        - GCC=7
+sudo: false
```

Comment:
`sudo: false` lets us use the container based builds which are faster to start (and execute usually), but limits what software we can install via package management